tasking/pulp/tasking/tasks.py

```diff
@@ -54,10 +52,10 @@ def _queue_reserved_task(name, task_id, resource_id, inner_args, inner_kwargs):
 
     :param name:          The name of the task to be called
     :type name:           basestring
-    :param inner_task_id: The UUID to be set on the task being called. By providing
+    :param task_id:       The UUID to be set on the task being called. By providing
```

Comment:
Oh you are fixing a consistency problem, but can the parameter itself be renamed from `task_id` -> `inner_task_id`. It isn't actually the task_id of the _queue reserve resource itself.